components/post_view/post_list_virtualized/post_list_virtualized.jsx

```diff
@@ -305,6 +307,12 @@ export default class PostList extends React.PureComponent {
         });
     };
 
+    toggleIsNewMessageLineReached = () => {
```

Comment:
Somewhat nitpicking, but maybe we could call this `onNewMessageLineReached`? The name toggle makes it sound a bit like we're going to be turning this state on and off when you're reading the code for `onItemsRendered`